cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql

```diff
@@ -2,3 +2,6 @@
 -- Schema cleanup from 5.3.6 to 5.3.7;
 --
 
+DROP TABLE IF EXISTS `cloud`.`snapshot_policy`;
```

Comment:
I thought this was going to be included on a separate pull request with an upgrade path.
Is it correct to put it in a `cleanup` script? I think I'm missing something.